pom.xml

```diff
@@ -237,6 +237,19 @@
     <error-prone.version>2.24.0</error-prone.version>
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
+    <error-prone.configuration-args>
+      <!--
+      Error Prone Support configuration:
+      -->
+      -Xep:JUnitClassModifiers:OFF
+      -Xep:JUnitMethodDeclaration:OFF
+      <!-- Check is not configurable yet. -->
+      -Xep:LexicographicalAnnotationListing:OFF
```

Comment:
in issue I see 
>please activate:
[LexicographicalAnnotationAttributeListing](https://error-prone.picnic.tech/bugpatterns/LexicographicalAnnotationAttributeListing/)

please share a reason.

Reply:
Correct, note the slight difference, we want to enable LexicographicalAnnotation**Attribute**Listing, but here I disabled the _LexicographicalAnnotationListing_. That simply sorts annotations lexicographically, which is not that desirable in most cases.

Reply:
@rnveach , @nrmancuso , do we want  to try sorting of annotations? 
I do think we have much of them, 1-2 is max on method I think

Reply:
We can disable it for now, and enable it in separate PR.
This is not a blocker to merge this PR

Reply:
When we introduce new tools, we usually suppress all one by one , explicitly and in next PRs, enable one by one.

Reply:
> @rnveach , @nrmancuso , do we want to try sorting of annotations?

Sorry for not checking this one with you guys, I didn't think too much about it as we don't use it internally. If there aren't that many annotations, it might be a nice addition.

> When we introduce new tools, we usually suppress all one by one , explicitly and in next PRs, enable one by one.

Do you mean that we should split the _current_ PR into multiple PRs? It's fine by me if you prefer this.

In any case, for next PRs I will keep it strictly to enabling **one** check per PR.

Reply:
No split for now, let's merge as is.

In next PRs we can use this approach, if convenient.

Reply:
In next PRs we can reuse https://github.com/checkstyle/checkstyle/tree/master/config/error-prone-suppressions to activate rule and simply put all/controversial violations to suppression and deal with them separately. At least nobody will merge a code to introduce more violations, there are a lot concurrent PRs now.

Reply:
As for annotation sorting, I prefer to have simple `@Blah` above all annotations like `@Blah(value={x, y, x})`, I do not care so much about lexicographical order.

Reply:
Ordering usually becomes matter, if there are numerous annotations.

@nrmancuso , do you mean you like piramid style of annotations? The most parametrized the bottom

Reply:
> "Pyramid"

Yes! This is a good way to describe it. I feel like the basic `@Blah` annotations sort of get lost, especially in Spring Boot applications.

Reply:
Ok, good to know, but this discussion should not block this PR, please review PR to let us unleash disablement of some rules. We can reconsider disablement in separate PR.